lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -325,6 +331,14 @@ def main():
                     image_table['image'][ii] = image
                     image_table['peak_time'][ii] = peak_time
 
+                    # use CatB pedestals to estimate the picture threshold
+                    if args.pedestal_cleaning:
+                        ped_charge_mean_pe = catB_pedestal_mean[calib_idx][selected_gain, pixel_index]
+                        ped_charge_std_pe = catB_pedestal_std[calib_idx][selected_gain, pixel_index]
+                        threshold_clean_pe = ped_charge_mean_pe + sigma * ped_charge_std_pe
+                        threshold_clean_pe[unusable_pixels] = pic_th
+                        picture_th = np.clip(pedestal_thresh, pic_th, None)
```

Comment:
I am not sure I am following this part. I expected something like:

```python
threshold_clean_pe[unusable_pixels] = max(threshold_clean_pe)
picture_th = np.clip(threshold_clean_pe, pic_th, None)
```

As it is done in `get_threshold_from_dl1_file`.

Reply:
the unusable pixels are set to zero in the Cat-B process, so in principle the pic_th is enough to cut them...